src/utils.js

```diff
@@ -0,0 +1,12 @@
+/** @const {String} ERROR_TOKEN Constant token used to mark that the operation failed */
+export const ERROR_TOKEN = "ERROR";
+
+/**
+ * Creates the standard returned response in case of operation failure
+ * 
+ * @param {Number} errorCode The error code to be included in the response
+ * @returns An array containing the ERROR_TOKEN and the error code
+ */
+export function errorResponse(errorCode) {
+    return [ERROR_TOKEN, errorCode];
+}
```

Comment:
It is not clear to me why you need to add this `ERROR_TOKEN` thing.

Reply:
It’s an element that marks that the operation failed. In my opinion, checking for one error token is more elegant than making arbitrary checks like “is returned element an array” or “is the size of the returned array equal to 1”